content/docs/1.10.1/important-notes/_index.md

```diff
@@ -38,6 +41,101 @@ For the full release note, see [here](https://github.com/longhorn/longhorn/relea
+
+backingimagedatasources.longhorn.io: ["v1beta2"]
+backingimagemanagers.longhorn.io: ["v1beta2"]
+backingimages.longhorn.io: ["v1beta2"]
+backupbackingimages.longhorn.io: ["v1beta2"]
+backups.longhorn.io: ["v1beta2"]
+backuptargets.longhorn.io: ["v1beta2"]
+backupvolumes.longhorn.io: ["v1beta2"]
+engineimages.longhorn.io: ["v1beta2"]
+engines.longhorn.io: ["v1beta2"]
+instancemanagers.longhorn.io: ["v1beta2"]
+nodes.longhorn.io: ["v1beta2"]
+orphans.longhorn.io: ["v1beta2"]
+recurringjobs.longhorn.io: ["v1beta2"]
+replicas.longhorn.io: ["v1beta2"]
+settings.longhorn.io: ["v1beta2"]
+sharemanagers.longhorn.io: ["v1beta2"]
+snapshots.longhorn.io: ["v1beta2"]
+supportbundles.longhorn.io: ["v1beta2"]
+systembackups.longhorn.io: ["v1beta2"]
+systemrestores.longhorn.io: ["v1beta2"]
+volumeattachments.longhorn.io: ["v1beta2"]
+volumes.longhorn.io: ["v1beta2"]
+```
+
+All CRDs should contains only `"v1beta2"` before v1.10.x upgrade.
+
+With these steps completed, the Longhorn CRD upgrade process should now proceed without issues.
+
+### Troubleshoot CRD Migration Failure After Upgrading Longhorn v1.10.x
+
+If these CRs are not fully migrated to `v1beta2`, the Longhorn Manager may fail to operate correctly. A common error message indicating this issue looks like:
+
+```
+Upgrade failed: cannot patch "backingimagedatasources.longhorn.io" with kind CustomResourceDefinition: CustomResourceDefinition.apiextensions.k8s.io "backingimagedatasources.longhorn.io" is invalid: status.storedVersions[0]: Invalid value: "v1beta1": missing from spec.versions; v1beta1 was previously a storage version, and must remain in spec.versions until a storage migration ensures no data remains persisted in v1beta1 and removes v1beta1 from status.storedVersions
+```
+
+To fix this issue, it require force downgrade Longhorn to original v1.9.x version. If the Longhorn is installed using `kubectl`, it is required to patch the `current-longhorn-version` before downgrade:
+
+```bash
+# attaching annotation to allow patching current-longhorn-version
+kubectl patch settings.longhorn.io current-longhorn-version -n longhorn-system --type=merge -p='{"metadata":{"annotations":{"longhorn.io/update-setting-from-longhorn":""}}}'
+# temporarily override current version to allow old version installation
+kubectl patch settings.longhorn.io current-longhorn-version -n longhorn-system --type=merge -p='{"value":"v1.9.0"}'
+```
+
+After modified `current-longhorn-version`, install the original v1.9.x Longhorn version.
+
+To Helm chart installation, downgrade is allowed with [`preUpgradeChecker.upgradeVersionCheck`](https://github.com/longhorn/longhorn/tree/v1.9.x/chart#other-settings) disabled.
+
+The Longhorn system is not fully downgraded with these steps. The major functions are not workable at this moment, for example, the volume scheduling. The only operational action avaiable is to [migrate the CRDs manually](#recommended-action-before-upgrading-longhorn-to-v110x), then upgrade to v1.10.x.
```

Comment:
After the downgrade, the Longhorn system enters an intermediate stage where major functions, such as volume scheduling, are not operational. The purpose of this stage is solely to migrate the legacy v1beta1 CRs to `v1beta2`. The only supported action at this point is to [manually migrate the CRDs](https://github.com/longhorn/website/pull/1211/files#recommended-action-before-upgrading-longhorn-to-v110x)
 and then proceed with the upgrade to v1.10.x.